osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -151,5 +172,53 @@ public void UpdateBeatmap(WorkingBeatmap beatmap)
                 Add(d);
             });
         }
+
+        private string GetBPMRange(WorkingBeatmap beatmap)

```

Comment:
Why are you passing a working beatmap in here? You can pass in a way lower level object.